test/cli/autogen_print_rbi/test.out

```diff
@@ -108,13 +108,6 @@ requires: []
  loc=test/cli/autogen_print_rbi/b.rbi:4
  is_defining_ref=0
 [ref id=2]
- scope=[FromRBI]
- name=[T]
- nesting=[[FromRBI]]
- resolved=[T]
- loc=test/cli/autogen_print_rbi/b.rbi:4
- is_defining_ref=0
-[ref id=3]
```

Comment:
Can we check with someone who works on autogen whether this is going to cause problems for any of their tools? cc @ngroman

Reply:
I'm not worried about this. The tooling that depends on this only is concerned with non-gem code.